src/chat_do.py

```
@@ -0,0 +1,337 @@
+                    "ws": ws,
+                    "user_id": user_id,
+                    "display_name": display_name,
+                }
+            except Exception as exc:
+                print(f"[ChatDO.__init__.restore] error={exc!r}")
+
+        self.ctx.setWebSocketAutoResponse(
+            WebSocketRequestResponsePair.new("ping", "pong")
+        )
+
+    async def _load_history(self, classroom_id: str):
```

Ruff (0.15.18)

[warning] Missing return type annotation for private function `_load_history`

Add return type annotation: `None`

(ANN202)